spec.html

```diff
@@ -38200,7 +38200,7 @@
         </emu-alg>
 
         <emu-annex id="sec-runtime-semantics-characterrangeorunion-abstract-operation" aoid="CharacterRangeOrUnion">
-          <h1>Runtime Semantics: CharacterRangeOrUnion Abstract Operation</h1>
+          <h1>Runtime Semantics: CharacterRangeOrUnion ( _A_, _B_ )</h1>
```

Comment:
It occurs 166 times in section-headers, but only 3 times otherwise, so it's both common and uncommon.